tomcat/osfamilymap.yaml

```diff
@@ -14,14 +14,15 @@ Debian:
   user: tomcat8
   group: tomcat8
   java_home: /usr/lib/jvm/default-java
+  # Use OS defaults:
+  catalina_base:
```

Comment:
Use empty string for yaml . `catalina_base: ''`

Reply:
Thanks, I pushed new changes according to your comment.